guides/it/laptop-security.md

```diff
@@ -0,0 +1,70 @@
+# Laptop Security
+In order to reach the standards for [Cyber Essentials Plus](https://www.ncsc.gov.uk/cyberessentials/overview) our laptops must meet the following requirements. 
+
+## Running a supported OS
+The operating system must be a current, supported version, that continues to receive security updates.
+
+### Linux (Ubuntu)
+Check [Ubuntu releases](https://wiki.ubuntu.com/Releases) to ensure that your version is supported.
+
+## User accounts
+The user account that you use on a day to day basis must be a standard non-administrator account. If you require administrator rights to do your job, you can create a separate admin user and switch to this when required.
+
+### Linux (Ubuntu)
+A standard account is not on the `sudoers list`, so in order to run commands that require `sudo` access you can temporarily switch to an admin account, run the `sudo` command then logout of the admin user:
+
+```bash
+su - [admin account]
+sudo [command]
+exit
+```
+
+## Account locking
+User accounts should be configured to lock after a maximum of 10 failed login attempts.
+
+### Linux (Ubuntu)
+To enable account locking on Ubuntu add the following line to `/etc/pam.d/common-auth`:
+
+```
+auth required pam_tally2.so onerr=fail deny=6 unlock_time=300 audit even_deny_root root_unlock_time=600
```

Comment:
The documentation says 10 failed attempts above, but this config would lock after 6 (`deny=6`) this might cause some confusion.

Reply:
yeah, good point - 10 is the maximum allowed, but that felt like a lot, so I opted for 6.  We probably need a Made Tech policy on this, but I'll clarify for now.